ui/index.d.ts

```diff
@@ -1174,6 +1177,11 @@ export declare namespace DialogResponses {
  */
 export declare function registerUIExtensionVariables(extVars: UIExtensionVariables): void;
 
+/**
+ * Call this to register the experimentation service adapter
+ */
+export declare async function registerExperimentationService(ctx: ExtensionContext, targetPopulation?: TargetPopulation): Promise<void>;
```

Comment:
Lets add some docs for `targetPopulation`. Maybe:
> Optional parameter to override default population ("Insiders" if extension version includes "alpha", otherwise "Public")